inc/remove-editor-settings.php

```diff
@@ -37,3 +37,60 @@ function block_directory() {
     }
 }
 add_action( 'admin_init', __NAMESPACE__ . '\block_directory' );
+
+/**
+ * Remove layout support, which means removing inline styles.
+ *
+ * @return void
+ */
+function remove_layout() {
+    // Allow filtering layout support.
+    if ( \apply_filters( 'meom_dodo_remove_layout_support', true ) ) {
+        remove_filter( 'render_block', 'wp_render_layout_support_flag', 10, 2 );
```

Comment:
Does this and `render_utility_classes` need some kind of check if WP 6.0 is used or is it ok to do these for every version?

Reply:
Hmmm not sure. `wp_render_layout_support_flag` is probably only in WP 6 so this wouldn't do anything before that.

I'll test with WP 5.*.

Reply:
That function is from 5.8. Tested with WP 5.7. and nothing broke.

But still not sure what would be correct check to be 100% there is no conflict for some old projects.

In that sense these can also be in Kala since that's always used in the latest WP when staring a project.